src/cpp/dynamic-types/TypeObject.cpp

```diff
@@ -9750,3 +7196,3433 @@ size_t to_size_t(
 } // namespace types
 } // namespace fastrtps
 } // namespace eprosima
+
+namespace eprosima {
```

Comment:
Same suggestion. Find attached the generated code from the [TypeObject.idl](https://github.com/eProsima/Fast-DDS/blob/bugfix/type-object/builtin-annotations/include/fastrtps/types/TypeObject.idl)
[TypeObjectCdrAux.txt](https://github.com/eProsima/Fast-DDS/files/12609143/TypeObjectCdrAux.txt)

Reply:
We are not sure the current types are equal that the defined one in idl files. I would keep the current code.